src/Dinkbit/Notifyme/Gateways/HipChat.php

```diff
@@ -193,13 +189,17 @@ public function jsonError($rawResponse)
      *
      * @param string $string
      *
-     * @return bool
+     * @return bool|array
      */
     protected function isJson($string)
```

Comment:
That's right it's not returning a bool only anymore.. `validateJson()` or `isValidJson`?